{"name": "Проверка появления ошибки при заполнении невалидного логина/пароля и возвращения правильного код ответа", "status": "passed", "attachments": [{"name": "stdout", "source": "ca1a5853-f51d-49bc-beb6-b1a22c9cca22-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "login", "value": "['grbplfdfjz', 'ziysvycxar', 'xqagkodczm']"}, {"name": "password", "value": "'123456'"}], "start": 1706799476894, "stop": 1706799477215, "uuid": "7d35ed58-2b37-4030-b9ba-b966aa539980", "historyId": "7dacd89f94894513b326e5d591e9006d", "testCaseId": "1948f0443281776f830c6ca14962e75a", "fullName": "tests.test_courier_auth.TestAuth#test_return_error_message_with_invalid_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_auth"}, {"name": "subSuite", "value": "TestAuth"}, {"name": "host", "value": "MacBook-Air-Kamilla.local"}, {"name": "thread", "value": "30509-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_auth"}]}